scripts/main/build.js

```diff
@@ -101,9 +103,14 @@ build.album = function (data, disabled = false) {
 
 	let html = lychee.html`
 			<div class='album ${disabled ? `disabled` : ``} ${data.is_nsfw && lychee.nsfw_blur ? `blurred` : ``}'
+				id='album-${data.id}'
 				data-id='${data.id}'
```

Comment:
Same here. No need to add the `id` because the `<div>` already contains this info in `data-id` and in `class`.